src/ResourceManagement/Search/AdminKeysImpl.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+namespace Microsoft.Azure.Management.Search.Fluent
```

Comment:
@milismsft Sorry for the confusion; My question wasn't about this class in particular, it was about all these changes. I maintain the Search SDKs in the main branches (vs17dev & psSdkJson6) but I've been ignoring all things Fluent up until now. Is this code ever going to be merged back into the mainline branches, or is it permanently separate?